main.tf

```diff
@@ -118,5 +121,5 @@ resource "aws_elasticache_cluster" "default" {
   apply_immediately            = var.apply_immediately
   preferred_availability_zones = slice(var.availability_zones, 0, var.num_cache_nodes)
   maintenance_window           = var.maintenance_window
-  tags                         = module.labels.tags
+  tags                         = merge(var.tags, module.labels.tags)
```

Comment:
we have a variable in labels called extra_lables for extra labels please use this